src/main/java/org/assertj/core/internal/Maps.java

```diff
@@ -774,7 +776,6 @@ public void assertHasSameSizeAs(AssertionInfo info, Map<?, ?> map, Map<?, ?> oth
                                            @SuppressWarnings("unchecked") Map.Entry<? extends K, ? extends V>... entries) {
     doCommonContainsCheck(info, actual, entries);
     if (actual.isEmpty() && entries.length == 0) return;
-    failIfEmpty(entries);
```

Comment:
apply the same check as commented before

Reply:
Done